src/xdist/workermanage.py

```diff
@@ -236,10 +236,10 @@ def shutting_down(self):
     def setup(self):
         self.log("setting up worker session")
         spec = self.gateway.spec
-        args = self.config.args
+        args = sys.argv[1:]
```

Comment:
that one will be incorrect on direct invocations of `pytest.main` and in other places

Reply:
Sure. 👍